presto-kudu/bin/run_kudu_tests.sh

```diff
@@ -53,14 +38,16 @@ function cleanup_docker_container() {
   docker-compose -f "${DOCKER_COMPOSE_LOCATION}" down
 }
 
-
 start_docker_container
 
 # run product tests
 pushd ${PROJECT_ROOT}
+# sleep to allow cluster be up
+sleep 15s
```

Comment:
is there a way to run some command to see if cluster is up and running? For presto we typically run `SELECT 1`.

Reply:
the sleep its not needed but will log to many error on log waiting for consensus between masters.
this could cause travis to fail with log exceed